src/main/scala/gemmini/Controller.scala

```diff
@@ -263,12 +266,21 @@ class GemminiModule[T <: Data: Arithmetic, U <: Data, V <: Data]
   }
   */
 
-  load_controller.io.cmd.valid := rob.io.issue.ld.valid
+  val load_cmd_ready = load_controller.map(_.io.cmd.ready)
+  rob.io.issue.ld.ready := load_cmd_ready.reduce(_||_)
+  for(d <- 0 until num_data_controller){
```

Comment:
I think this should be arbitrated so that only one load controller gets the request, right?

Otherwise, we'll make the same read request many times